modules/installation-creating-aws-control-plane.adoc

```diff
@@ -217,6 +223,17 @@ You need the name of this stack if you remove the cluster.
 YAML file that you saved.
 <3> `<parameters>` is the relative path to and name of the CloudFormation
 parameters JSON file.
++
+.Example output
+[source,terminal]
+----
+arn:aws:cloudformation:us-east-1:269333783861:stack/cluster-control-plane/21c7e2b0-2ee2-11eb-c6f6-0aa34627df4b
+----
++
+[NOTE]
+====
+A stack that represents three control plane nodes is created when the CloudFormation template is launched.
```

Comment:
Can this be in active voice? For example:
"At launch, the CloudFormation template creates a stack that represents three control plane nodes." (You might even be able to drop "at launch".)

Reply:
Thank you for highlighting the need for active voice in this note. I have implemented part of your suggestion, but have minimised it slightly:

"The CloudFormation template creates a stack that represents three control plane nodes."